internal/app/connectconformance/testsuites/errors.yaml

```diff
@@ -412,6 +412,31 @@ testCases:
         responseTrailers:
         - name: x-custom-trailer
           value: ["bing"]
+# Client Stream Tests -----------------------------------------------------------
```

Comment:
Admittedly a drive-by change. There were no client stream error tests in this file.